frontend/src/ui/Blocks/Series.tsx

```diff
@@ -166,24 +168,29 @@ const ReadySeriesBlock: React.FC<ReadyProps> = ({
     const timeMs = (event: Event) =>
         new Date(event.syncedData?.startTime ?? event.created).getTime();
 
+    const sortEvents = (a: Event, b: Event) =>
+        match(eventOrder, {
+            "NEW_TO_OLD": () => timeMs(b) - timeMs(a),
+            "OLD_TO_NEW": () => timeMs(a) - timeMs(b),
+            "A-Z": () => a.title.toLowerCase() > b.title.toLowerCase() ? 1 : -1,
+            "Z-A": () => a.title.toLowerCase() > b.title.toLowerCase() ? -1 : 1,
```

Comment:
This should use `localeCompare` as the expected string order actually depends on the locale: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare

I think this also already deals with casing (in that it ignores it, if the locale says so)

Reply:
You still need to pass in the correct language thats set in Tobira. You can compare `sortRealms` in `routes/util.tsx`. There we also create an `Intl.Collator` as MDN says thats better for performance: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare#performance